Azure/sonic-buildimage
# Platform/cisco-8000 module for sonic-buildimage

#### Why I did it
Update Makefile, so it does the following: 
For a given platform, verify if platform/checkout/<platform>.ini exists and hence run the platform/checkout/template.j2. This allows platform code to be checked out during the 'make configure' stage. 

#### How I did it
git clone git@github.com:Azure/sonic-buildimage.git
mkdir platform/cisco-8000

make init
make configure PLATFORM=cisco-8000
make all

#### How to verify it

Ensure, the platform/checkout/<platform>.ini file refers to the correct github page with the correct version tag that can be downloaded. 

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
